lib/simple_forms_api_submission/metadata_validator.rb

```diff
@@ -2,27 +2,27 @@
 
 module SimpleFormsApiSubmission
   class MetadataValidator
-    def self.validate(metadata)
+    def self.validate(metadata, zip_code_is_us_based: true)
```

Comment:
code looks good to me but have a question about this part - what determines if the zip code is us based? because that data doesn't exist in the meta data right? the zip alone is not enough to determine that?

Reply:
Correct, it will be up to whatever calls the MetadataValidator to tell it whether the address/zip code is US or not. I want to leave the validator as ignorant of the incoming form data as possible. The zip code alone is insufficient because the US is apparently not the only country to use a 5-digit format (Germany does too).

Reply:
got it, that makes sense, a couple followup questions:
1) I just wasn't sure which code was responsible for determining that value if it's us based or not. how do they know which value to use? does that come from the form data?
2) right now it's defaulting to true, does that mean we can't send their actual zip code if they're in canada and we have to instead send 00000? just making sure that's intentional

Reply:
1. Yes it'll ultimately have to come from form data. Wherever we collect zip we ought to also collect Country so it should be doable to pull that piece out to answer the boolean for the metadata validator.
2. With this change, a form with an address in Canada can send either `00000` or the real zip code. Either way it'll turn into `00000` for the metadata (assuming the boolean is set to `false`). Does that make sense?

Thanks for the followups!

Reply:
Got it that make sense. A few followup questions, but all this sounds good to me- approving ✅

Is something calling this right now? you make it sound like the consumer side isn't implemented yet. just wanted to make sure if something is not passing that second parameter, it's okay to assume that it's going to be a us address.